kuiper-tests/GameTimeShould.cs

```diff
@@ -1,38 +1,77 @@
 using System;
+using System.Threading;
+using Kuiper.Domain;
+using Shouldly;
 using Xunit;
 
-namespace Kuiper.Tests.Unit.Systems
+namespace Kuiper.Tests
 {
     [Collection("Sequential")]
-    public class GameTimeStaticShould
+    public class GameTimeShould
     {
+        private readonly DateTime _gameTimeEpoch = new DateTime(2078, 1, 8);
+        private readonly long _tickAccelerationConstant = 7; //1 real day is 7 game days
+        
         [Fact]
-        public void ThrowExceptionIfRealStartTimeNeverSet()
+        public void Return_GameTime_Epoch_For_A_Real_Seed()
```

Comment:
Consider adopting the Roy Osherove standard of test naming (because writing with `_` sucks (atleast it always hurts in my hand).

 [UnitOfWork_StateUnderTest_ExpectedBehavior]
 `GameTime_WithSeed_ReturnsEpoch` or something to that effect

Reply:
Hmm 🤔 

Personally, I like the Should convention: `StructureUnderTest_Should_ExpectedBehavior_When_StateUnderTest` (I kinda got that from the class naming (the `StructureUnderTest_Should` part)) 

@mifriis do you have a [naming convention](https://dzone.com/articles/7-popular-unit-test-naming) in place, you would like to be used?

Reply:
I think he has multiple naming conventions going on! :)